File: app/src/main/java/com/modori/kwonkiseokee/AUto/showPhotoOnly.java
package com.modori.kwonkiseokee.AUto;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.bumptech.glide.Glide;
import com.github.chrisbanes.photoview.PhotoView;

public class showPhotoOnly extends AppCompatActivity {

    PhotoView photoView;
    String regularUrl;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.show_photoonly);

        Intent intent = getIntent();
        regularUrl = intent.getStringExtra("photoUrl");
        photoView = findViewById(R.id.photoView);

        Glide.with(this).load(regularUrl).into(photoView);





    }
}

File: app/src/main/java/com/modori/kwonkiseokee/AUto/ShowPhotoOnly.kt
package com.modori.kwonkiseokee.AUto

import android.os.Bundle
import android.util.Log
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.bumptech.glide.Glide
import kotlinx.android.synthetic.main.show_photoonly.*
import java.lang.Error

class ShowPhotoOnly : AppCompatActivity(){
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.show_photoonly)

        try{
            val intent = intent
            val url = intent!!.getStringExtra("photoUrl")
            Glide.with(this).load(url).into(photoView)
        }catch (e:Error){
            Toast.makeText(this,"죄송합니다. 오류가 발생했습니다.\n ${e.message}",Toast.LENGTH_SHORT).show()
            Log.e("ShowPhotoOnly", e.message!!)
            finish()
        }

    }
}